R/conversion.R

```diff
@@ -336,3 +336,35 @@ dim.pandas.core.frame.DataFrame <- function(x) {
   else
     pandas_shape(x)
 }
+
+# Conversion between `Matrix::dgCMatrix` and `scipy.sparse.csc.csc_matrix`.
+# Scipy CSC Matrix: https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csc_matrix.html
+
+#' @export
+r_to_py.dgCMatrix <- function(x, convert = FALSE) {
```

Comment:
I think we might need to import `Matrix` if we're going to define S3 methods for classes exported by that package

Reply:
Should I move Matrix to `Imports` in DESCRIPTION and then remove the `require` statement?